File: UNIDAD 3/C#/U3_P12.cs
using System;
namespace U3_P12
{
    internal class U3_P12
    {
        static void Main(string[] args)
        {
            int c, total, num;//2
            Console.Write("Escribe el total de numeros a procesar: ");//3
            total = Convert.ToInt32(Console.ReadLine());//4
            for (c = 1; c <= total; c++)
            { //5
                Console.WriteLine("Escribe numero: "); //5a
                num = Convert.ToInt32(Console.ReadLine()); //5b
                if (num % 2 == 0) //5c
                    Console.WriteLine("Par");
           else //5d
                    Console.WriteLine("Impar");

            }


            
         
        }
    }
}

File: UNIDAD 3/JAVA/U3_P12.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p12;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class U3_P12 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in); //1
        int c, total, num;//2
        System.out.print("Escribe el total de numeros a procesar: ");//3
        total = sc.nextInt();//4
        for (c=1; c<=total; c++){ //5
            System.out.println("Escribe numero: "); //5a
            num = sc.nextInt(); //5b
           if (num % 2 == 0) //5c
               System.out.println("Par");
           else //5d
               System.out.println("Impar");
        
        }
    }
}
